.github/workflows/release.yaml

```diff
@@ -217,12 +254,56 @@ jobs:
           github-tag-template: ${{ inputs.github-tag-template }}
           assets: ${{ inputs.assets }}
 
-      - name: write-release-notes-to-file
+      - name: write-filtered-release-notes-to-file
         if: ${{ inputs.slack-channel-id != '' }}
+        shell: python
         run: |
-          cat <<"EOF" > release-notes.md
-          ${{ steps.release-notes.outputs.full-release-notes }}
-          EOF
+          import yaml
```

Comment:
generally-speaking, I think the filtering should be done in `release-notes`-action.

-> pass filtering-arguments to release-notes-action, propagate whatever release-notes action returns. This will also allow us to convey such parameters differently (e.g. via cfg-file in repository, or via cfg-artefact)